src/main/resources/db/migration/postgresql/V1_10__2055_Topic_Column_On_Workstack_Team_Queue.sql

```diff
@@ -0,0 +1,16 @@
+DROP VIEW IF EXISTS stage_data;
+
+CREATE OR REPLACE VIEW stage_data AS
+    SELECT c.reference AS case_reference, c.type AS case_type, c.data as data, c.created AS case_created, s.*, t.text AS case_assigned_topic
+    FROM stage s
+            JOIN active_case c ON s.case_uuid = c.uuid
+            LEFT JOIN casework.topic t ON c.primary_topic_uuid = t.uuid;
+
+DROP VIEW IF EXISTS active_stage_data;
```

Comment:
Do we need to drop the views, the create or replace view will do this for you? views act as virtual tables and thus from my knowledge don't hold info.